### ISSUE TITLE
Can't reproduce. Complete calendar task "create a event of laundry" loops forever.

### ISSUE THREAD

**Issue by leosol** [MobileLLM/AutoDroid#12]
After configuring openai Key and installing all the required dependencies, verything worked almost as expected, but unfortunatelly this sample task did not finish. Was it expected to complete?
[autodroid.log](https://github.com/MobileLLM/AutoDroid/files/14393075/autodroid.log)

**Comment by wenh18** [comment]
Yes, The current implementation is not good at determining task completion. We mentioned it in the failure case discussion in Section 6.2 and 'Note that we do not include the completion determination step for clear comparison...' in the Task Completion Rate section.
